audplot/core/api.py

```diff
@@ -146,6 +149,11 @@ def confusion_matrix(
 
             >>> confusion_matrix(truth, prediction, labels=['A', 'B', 'C', 'D'])
 
+        .. plot::
+            :context: close-figs
+
+            >>> confusion_matrix(truth, prediction, label_aliases={'A': 0, 'B': 1, 'C': 2})
```

Comment:
Would it make sense that e change the docstring examples to the more likely use cases where we have encoded labels `0, 1, 2` which we then map to strings `A, B, C`?

Reply:
Good idea, I changed it accordingly.